app/services/users/delete_activity.rb

```diff
@@ -12,6 +12,33 @@ def call(user)
       user.mentions.delete_all
       user.badge_achievements.delete_all
       user.github_repos.delete_all
+
+      user.access_grants.delete_all
```

Comment:
I had to add a lot of deletions to `DeleteActivity` because otherwise, they will keep hanging out in the database.